platform/security/security-oidc-realm/src/main/java/org/codice/ddf/security/oidc/realm/OidcRealm.java

```diff
@@ -34,53 +39,118 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(OidcRealm.class);
 
   private List<String> usernameAttributeList;
+  private OidcHandlerConfiguration oidcHandlerConfiguration;
 
   /** Determine if the supplied token is supported by this realm. */
   @Override
   public boolean supports(AuthenticationToken token) {
-    boolean supported =
-        token != null && token.getCredentials() != null && token instanceof OidcAuthenticationToken;
-
-    if (supported) {
-      LOGGER.debug("Token {} is supported by {}.", token.getClass(), OidcRealm.class.getName());
-    } else if (token != null) {
-      LOGGER.debug("Token {} is not supported by {}.", token.getClass(), OidcRealm.class.getName());
-    } else {
-      LOGGER.debug("The supplied authentication token is null. Sending back not supported.");
+    if (token instanceof SessionToken) {
```

Comment:
📘 If the `OidcHandler` receives a session token, the session token doesn't have a web context, so we cannot resolve credentials. A valid Id token must therefore be present within the token.